README.md

```diff
@@ -169,9 +177,23 @@ every reference to this item.
 > [!NOTE]
 > (Temporary limitation)
 >
-> At the moment, definitions that belong to *some* sections of the
-> data division, and some references to data items, are ignored by the
-> extension.  They will be covered by the first stable release.
+> In addition to limitations mentioned in [Go to
+> Definition](#go-to-definition), references that occur in
+> `EXEC`/`END-EXEC` blocks are also not taken into account yet.
+
+### Reference Information
+
+The extension shows inline reference information above definitons of
+data items and elements of the procedure division.  
+The same limitations as for [Go to Definition](#go-to-definition)
+apply.
```

Comment:
please add a sscreenshot showing the feature here as well; I _guess_ that possibly means #354 can be closed

Reply:
Thanks indeed the screenshot is missing here.
#354 would probably be easy to implement (including the reference count in the generated information shown on hover), but it's not done yet.